services/blockchain-connector/shared/sdk/decoder.js

```diff
@@ -72,10 +73,16 @@ const decodeTransaction = (transaction) => {
 
 	const txParamsSchema = getTransactionParamsSchema(transaction);
 	const transactionParams = codec.decode(txParamsSchema, Buffer.from(transaction.params, 'hex'));
+	const nonEmptySignaturesCount = transaction.signatures.filter(s => s).length;
 	// TODO: Verify if 'computeMinFee' returns correct value
```

Comment:
Can we please remove the TODO since this is already being address here?

Reply:
Applied.